frontend/src/router.ts

```diff
@@ -102,7 +102,7 @@ export const router = new Router({
               component: () => import("@/views/TE/Task.vue"),
               meta: {
                 menuItem: false,
-                auth: { requireClaims: { TE: true, coordinator: true } }
+                auth: { roles: ["TE.checker", "TE.editor", "TE.coordinator"] }
```

Comment:
Are you sure we can't utilize Abilities in the router? I see it as specifying the action and subject in the meta, and then checking this in the navigation guard.

For example, for this case it may be `auth: { action: 'view', subject: 'TE/Task' }`.

Reply:
Yes I will look into refactoring it like that. Although should action be included? It is implied by having the subject.